lib/webpacker/runner.rb

```diff
@@ -9,9 +9,9 @@ def self.run(argv)
     def initialize(argv)
       @argv = argv
 
-      @app_path          = File.expand_path(".", Dir.pwd)
-      @node_modules_path = File.join(@app_path, "node_modules")
-      @webpack_config    = File.join(@app_path, "config/webpack/#{ENV["NODE_ENV"]}.js")
+      @app_path              = File.expand_path(".", Dir.pwd)
+      @node_modules_bin_path = ENV['WEBPACKER_NODE_MODULES_BIN_PATH'] || %[yarn bin]
```

Comment:
@tmaier Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
> @node_modules_bin_path = ENV['WEBPACKER_NODE_MODULES_BIN_PATH'] || %[yarn bin]
and it will fix styling rubocop rules.